t/test_helper.rb

```diff
@@ -58,15 +58,15 @@ def stub_github_api
     private
 
     def cdn
-      'https://cdn.rawgit.com'
+      'https://github.com'
```

Comment:
It feels a little odd for this to be `cdn` now. This is definitely the easy change to make everything Just Work™, but the mismatch here seems like it might be a little confusing later. It's not a show-stopper, but it certainly causes me some hesitation. Is this used directly from tests, or just locally here? If the latter, I'd be very tempted to just eliminate this method now.